Guide/src/dev_guide/tests/unit.md

```diff
@@ -1,13 +1,7 @@
 # Unit Tests
 
 ```admonish tip
-Note: We recommend using [cargo-nextest](https://nexte.st/) to run unit / VMM
-tests. It is a significant improvement over the built-in `cargo test` runner,
-and is the test runner we use in all our CI pipelines.
-
-You can install it locally by running: `cargo install cargo-nextest --locked`
-
-See the [cargo-nextest](https://nexte.st/) documentation for more info.
+{{ #include /_fragments/nextest_tip.md }}
```

Comment:
Did you *have* to put the fragment inside the admonish block here, at the consuming side? (Or did you do it this way so that consumers of the fragment can decide if it's in an admonishment or not).

Reply:
unfortunately, this is a limitation of how the current templating infrastructure works :(

I'm sure with a bit more custom preprocessing / hacking I could maybe get this working, but at a certain point, you gotta throw in the towel and bend to the tool you're using